### PR TITLE
Landin page

### PR DESCRIPTION
**Pull request by Developers-codz**
## Files to check 

`src/components/aside/Aside.jsx`
`src/components/category/Category.jsx`
`src/components/navbar/Navbar.jsx`
`src/context/aside-context.js`
`src/context/video-context.js`
`src/functions/get-categorised-video.js`
`src/page/home/Home.jsx`
`src/reducer/video-reducer.js`

Capture

Capture2

### Thank you for reviewing my PR :)

### CODE REVIEW

**Review by Maianki — APPROVED** [review]
Looks good to merge. 
Your code is very readable. You have abstracted your code wherever it was necessary so great work there. And UI looks nice too.

File: src/components/aside/Aside.jsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+            <li className={style.asideLink}>Home</li>
+          </NavLink>
+          <NavLink className="decor-none light-text" to="/playlist">
+            <li className={style.asideLink}>Playlist</li>
+          </NavLink>
+          <NavLink className="decor-none light-text" to="/history">
+            <li className={style.asideLink}>History</li>
+          </NavLink>
+          <NavLink className="decor-none light-text" to="/liked">
+            <li className={style.asideLink}>Liked</li>
```
Comment (ApurvaSawant11) [inline src/components/aside/Aside.jsx]: You can use `map` here since most of the code is repeating. Also, add icons, it will look good

**Review by ApurvaSawant11 — APPROVED** [review]
Looks good to me! Design also looks nice

Saw your Netlify Deploy log. Try this:

1. Go to your current site settings > Build & Deploy > Build settings > In Build Command, paste this as it is     `CI= npm run build`

2. Also add JWT_SECRET in environment variables, you'll need itf or authentication

**Comment by Developers-codz** [comment]
> Looks good to me! Design also looks nice
> 
> Saw your Netlify Deploy log. Try this:
> 
> 1. Go to your current site settings > Build & Deploy > Build settings > In Build Command, paste this as it is     `CI= npm run build`
> 2. Also add JWT_SECRET in environment variables, you'll need itf or authentication

**Comment by Developers-codz** [comment]
> > Looks good to me! Design also looks nice
> > Saw your Netlify Deploy log. Try this:
> > 
> > 1. Go to your current site settings > Build & Deploy > Build settings > In Build Command, paste this as it is     `CI= npm run build`
> > 2. Also add JWT_SECRET in environment variables, you'll need it for authentication

I deployed my dev branch. In build command I pasted `ci=false npm run build` it get deployed there its working as well [play-arts](https://play-arts.netlify.app/) the issue is in deploying this branch only just I don't know what is it.

File: src/components/aside/Aside.jsx (hunk 1 above)
Comment (Developers-codz) [reply src/components/aside/Aside.jsx]: Will map that thank you

**Pull request merged**